modules/standard/Regexp.chpl

```diff
@@ -849,9 +849,9 @@ record regexp {
       if pos < splitstart {
         // Yield splitted value
         if exprType == string then
-          yield text[pos+1..splitstart];
+          yield text[pos..splitstart-1];
         else 
-          yield text[(pos+1):int..splitstart:int];
+          yield text[pos:int..(splitstart-1):int];
```

Comment:
Not related to this PR but: we no longer need to cast byteIndex to int to use bytes. But this code probably predates that change. I can remove that check once this PR gets in.

Reply:
OK, so the idea would be to remove the conditional here and just use the same slice code in either case, is that right?  I can do that.

Reply:
Yep `yield text[pos..splitstart-1];` should do it

Reply:
Fixed, thanks!